app/assets/javascripts/evercookie/evercookie.js.erb

```diff
@@ -298,7 +297,7 @@ var <%= Evercookie.js_class %> = (function (window) {
             params.swliveconnect = "true";
             attributes.id        = "myswf";
             attributes.name      = "myswf";
-            swfobject.embedSWF("<%= asset_path('ec.swf') %>", "swfcontainer", "1", "1", "9.0.0", false, flashvars, params, attributes);
+            swfobject.embedSWF("<%= asset_url('ec.swf') %>", "swfcontainer", "1", "1", "9.0.0", false, flashvars, params, attributes);
```

Comment:
asset_url renders the protocol based on the request...but that won't work here as the js will be pre-compiled. 

I'll make a change.